### ISSUE TITLE
Active persisted queries for Apollo has an issue with the signature query.

### ISSUE THREAD

**Issue by michaelstaib** [ChilliCream/hotchocolate#1042]
Hot Chocolate expects the query lookup through the `id` field or the `namedQuery` field. Apollo Links on the other hand always puts the query hash into the extension map.

Signature Query:

```javascript
{
  operationName: 'MyQuery',
  variables: null,
  extensions: {
    persistedQuery: {
      version: 1,
      sha256Hash: hashOfQuery
    }
  }
}
```

Another example:

```json 
{
  "operationName": null,
  "variables": {},
  "extensions": {
    "persistedQuery": {
      "version": 1,
      "sha256Hash": "f3ec70c85b5ad66d3ef8d445a2097ac412b00221f76d83999042976e20eac85b"
    }
  }
}
```

https://blog.apollographql.com/improve-graphql-performance-with-automatic-persisted-queries-c31d27b8e6ea

https://github.com/apollographql/apollo-link-persisted-queries

**Comment by michaelstaib** [comment]
This issue is now fixed and integrated into 10.1.0-preview.4

**Comment by michaelstaib** [comment]
https://github.com/ChilliCream/hotchocolate/commit/afd4f59df5cf5aa9f4094324c4a2c8dc272fc824

**Comment by michaelstaib** [comment]
The fix is now integrated in release 10.1.0-preview.4

https://github.com/ChilliCream/hotchocolate/releases/tag/10.1.0-preview.4

And also we have integrated the fix into release 11.0.0-preview.5

https://github.com/ChilliCream/hotchocolate/releases/tag/11.0.0-preview.5
